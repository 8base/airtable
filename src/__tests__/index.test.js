const { convertSchema } = require('../')
const { AIRTABLE_SCHEMA } = require('../__fixtures__')

it('As a developer, I can convert Airtable schema to the 8base schema.', () => {
  expect(convertSchema(AIRTABLE_SCHEMA)).toMatchSnapshot()
})
