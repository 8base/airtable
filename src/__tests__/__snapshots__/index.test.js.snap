// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`As a developer, I can convert Airtable schema to the 8base schema. 1`] = `
Array [
  Object {
    "displayName": "Initiatives & Organizations",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Initiative Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fld8CqSXrBC2TwqVp",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "initiativeName",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Description",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fldFcGOl9XjlojA1C",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "description",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Image",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "flda30G0tIyE6TF02",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "image",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "URL",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldprKpRcNdQzOuYf",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "url",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Key People",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldSN17LxImuQcyhe",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "keyPeople",
        "relation": Object {
          "refFieldDisplayName": "Organizations",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "organizations",
          "refTable": Object {
            "id": "tblYvJ9X94Gq3OKju",
            "name": "people",
          },
          "refTableId": "tblYvJ9X94Gq3OKju",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Service Categories",
        "fieldType": "SWITCH",
        "fieldTypeAttributes": Object {
          "format": "CUSTOM",
          "listOptions": Array [
            "Peer-to-Peer",
            "Data Collection",
            "Debt Management",
            "Lending & Credit",
            "Payments",
            "Investment",
            "Determining Credit Worthiness",
            "Deposits & Savings",
            "Financial Counseling & Education",
            "Wealth Management",
            "Microfinance",
            "Financial Reporting",
            "Needs Assessment",
            "Lending and Credit",
            "Money Transfers",
            "Identity System",
            "Insurance",
            "Budgeting",
          ],
        },
        "id": "fldoYYNG7cBLoNMRe",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "serviceCategories",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Affiliated Initiatives & Organizations",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldzfxKazrHJgMJuo",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "affiliatedInitiativesOrganizations",
        "relation": Object {
          "refFieldDisplayName": "Affiliated Initiatives & Organizations (Self)",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "affiliatedInitiativesOrganizationsSelf",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Locations / Regions",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldyZvzKp1B6k6LFj",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "locationsRegions",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Documentation",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldU1sxKxpgsxK1ez",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "documentation",
        "relation": Object {
          "refFieldDisplayName": "Initiative or Org",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "initiativeOrOrg",
          "refTable": Object {
            "id": "tblLQvp37cl8wOUnj",
            "name": "documentation",
          },
          "refTableId": "tblLQvp37cl8wOUnj",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Objectives",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldgYEM8xP0hrwf2T",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "objectives",
        "relation": Object {
          "refFieldDisplayName": "Orgs / Projects / Products",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "orgsProjectsProducts",
          "refTable": Object {
            "id": "tbllsjYlfEMlFvAoZ",
            "name": "objectives",
          },
          "refTableId": "tbllsjYlfEMlFvAoZ",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Tech Types",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldzyHIpl9w64LncQ",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "techTypes",
        "relation": Object {
          "refFieldDisplayName": "Initiatives & Organizations",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "initiativesOrganizations",
          "refTable": Object {
            "id": "tblKhLxd8Q7hTEzpJ",
            "name": "technologyTypes",
          },
          "refTableId": "tblKhLxd8Q7hTEzpJ",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Twitter",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldRa3j4na4H2mXVp",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "twitter",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Blog / Updates / RSS",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldeVuhkyNxZhNgHM",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "blogUpdatesRss",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Key Partners & Sponsors",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldAiuKOnzWlMBCKA",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "keyPartnersSponsors",
        "relation": Object {
          "refFieldDisplayName": "Initiatives & Organizations",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "initiativesOrganizations",
          "refTable": Object {
            "id": "tbl7iuSfoHTVTf4Oo",
            "name": "partnerSponsorOrgs",
          },
          "refTableId": "tbl7iuSfoHTVTf4Oo",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Beneficiaries",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fld1DFB8pVcITjnbB",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "beneficiaries",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Global Goals",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fld5a32LaQzMhIRyi",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "globalGoals",
        "relation": Object {
          "refFieldDisplayName": "Initiatives & Organizations",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "initiativesOrganizations",
          "refTable": Object {
            "id": "tblAeFxeUN3ZU6DVQ",
            "name": "globalGoals",
          },
          "refTableId": "tblAeFxeUN3ZU6DVQ",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Type",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldsXJtoRaxeUATYV",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "type",
        "relation": Object {
          "refFieldDisplayName": "Initiatives & Organizations",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "initiativesOrganizations",
          "refTable": Object {
            "id": "tblNMtn84ZahQOqtM",
            "name": "initiativeType",
          },
          "refTableId": "tblNMtn84ZahQOqtM",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Last Updated",
        "fieldType": "DATE",
        "fieldTypeAttributes": Object {
          "format": "DATE",
        },
        "id": "flds8dNz6f5W29yxJ",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "lastUpdated",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Latitude",
        "fieldType": "NUMBER",
        "fieldTypeAttributes": Object {
          "currency": null,
          "format": "NUMBER",
          "maxValue": null,
          "minValue": null,
          "precision": 7,
        },
        "id": "fldXRoMnxcXhHfTiI",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "latitude",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Longitude",
        "fieldType": "NUMBER",
        "fieldTypeAttributes": Object {
          "currency": null,
          "format": "NUMBER",
          "maxValue": null,
          "minValue": null,
          "precision": 7,
        },
        "id": "fldkRApX7O2Xf5RiN",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "longitude",
        "relation": null,
      },
    ],
    "id": "tblwo8xeP7k63eosx",
    "name": "initiativesOrganizations",
  },
  Object {
    "displayName": "Objectives",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldOWWkYCVYuO07DA",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Notes",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fldfNp8G8dM9CQJQj",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "notes",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Attachments",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldra9dtX6pu6rfRx",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "attachments",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Orgs / Projects / Products",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldglL4BYRojobX93",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "orgsProjectsProducts",
        "relation": Object {
          "refFieldDisplayName": "Objectives",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "objectives",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
    ],
    "id": "tbllsjYlfEMlFvAoZ",
    "name": "objectives",
  },
  Object {
    "displayName": "Global Goals",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldqtNagEnwElReYX",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Notes",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fldbccdTliA25pHlt",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "notes",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Attachments",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldPCI1knjjGGKsKF",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "attachments",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Initiatives & Organizations",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldqkDPe3rwxLmwne",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "initiativesOrganizations",
        "relation": Object {
          "refFieldDisplayName": "Global Goals",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "globalGoals",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "URL",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldy76Y7FDhEEduao",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "url",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Image",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldVAkqC0FFguYKtn",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "image",
        "relation": null,
      },
    ],
    "id": "tblAeFxeUN3ZU6DVQ",
    "name": "globalGoals",
  },
  Object {
    "displayName": "Initiative Type",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldgV1im3yp5lCfSW",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Notes",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fldRzWMntOP6EVSEK",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "notes",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Initiatives & Organizations",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldrO9fTOTKnIVwlU",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "initiativesOrganizations",
        "relation": Object {
          "refFieldDisplayName": "Type",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "type",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "URL",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldpTrISR4pi8szuz",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "url",
        "relation": null,
      },
    ],
    "id": "tblNMtn84ZahQOqtM",
    "name": "initiativeType",
  },
  Object {
    "displayName": "People",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldbOosefqyHmqWeo",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Notes",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fldBv5eTWhOEsLOek",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "notes",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Images",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldgAP6maXaZXSuuz",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "images",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Organizations",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fld0t7ftaD4uG4jTK",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "organizations",
        "relation": Object {
          "refFieldDisplayName": "Key People",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "keyPeople",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Linkedin",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldrXWnMjIUplyg7D",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "linkedin",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Blogs / Updates",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldWon8jpx6PQk4tQ",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "blogsUpdates",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Twitter",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldN2L7hY614Yq2zk",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "twitter",
        "relation": null,
      },
    ],
    "id": "tblYvJ9X94Gq3OKju",
    "name": "people",
  },
  Object {
    "displayName": "Technology Types",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldrBwOE0IAyErABz",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Notes",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fld5VKutcIB2vMgVi",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "notes",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Attachments",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldnfWM6Okiqob73u",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "attachments",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Learn more",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldARWbboOxhIhfPX",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "learnMore",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Initiatives & Orgs",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldQoTKVSwdF6QEZ1",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "initiativesOrgs",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Service Categories",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldwnGsP8mV7vdFjy",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "serviceCategories",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Initiatives & Organizations",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fld67BTyjLbaZy0zO",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "initiativesOrganizations",
        "relation": Object {
          "refFieldDisplayName": "Tech Types",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "techTypes",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
    ],
    "id": "tblKhLxd8Q7hTEzpJ",
    "name": "technologyTypes",
  },
  Object {
    "displayName": "Documentation",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldewqnqW8ss3VRQP",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "URL",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldyOjNUDg0Gi9tV2",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "url",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Attachments",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldadA5oh2LESjrOD",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "attachments",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Initiative or Org",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fld3aFyPbKeis3iNE",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "initiativeOrOrg",
        "relation": Object {
          "refFieldDisplayName": "Documentation",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "documentation",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
      Object {
        "defaultValue": null,
        "displayName": "Image",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldAIM7bAzlOPyHch",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "image",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Description",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fld35HHl6rHy6T2KD",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "description",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Type",
        "fieldType": "SWITCH",
        "fieldTypeAttributes": Object {
          "format": "CUSTOM",
          "listOptions": Array [
            "Toolkits & Guides",
            "Code Repositories",
            "APIs",
            "Publications",
          ],
        },
        "id": "fldUmrcIHfaLIlgxE",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "type",
        "relation": null,
      },
    ],
    "id": "tblLQvp37cl8wOUnj",
    "name": "documentation",
  },
  Object {
    "displayName": "Partner & Sponsor Orgs",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldkaxCkaz8oB2na1",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Description",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fld4Tct3oRMV3J2Xl",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "description",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "URL",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldcMvfIWVmoZn3HQ",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "url",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Twitter",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldPAAGjs4OUToKcN",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "twitter",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Initiatives & Organizations",
        "fieldType": "RELATION",
        "fieldTypeAttributes": null,
        "id": "fldjvycf7zu25RQxv",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "initiativesOrganizations",
        "relation": Object {
          "refFieldDisplayName": "Key Partners & Sponsors",
          "refFieldIsList": true,
          "refFieldIsRequired": false,
          "refFieldName": "keyPartnersSponsors",
          "refTable": Object {
            "id": "tblwo8xeP7k63eosx",
            "name": "initiativesOrganizations",
          },
          "refTableId": "tblwo8xeP7k63eosx",
        },
      },
    ],
    "id": "tbl7iuSfoHTVTf4Oo",
    "name": "partnerSponsorOrgs",
  },
  Object {
    "displayName": "Data",
    "fields": Array [
      Object {
        "defaultValue": null,
        "displayName": "Name",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldbjkCnWsUbPB2rl",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "name",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Notes",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 65536,
          "format": "UNFORMATTED",
        },
        "id": "fldIJKlPEnfBfYImJ",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "notes",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Attachments",
        "fieldType": "FILE",
        "fieldTypeAttributes": Object {
          "format": "FILE",
        },
        "id": "fldkpsDVIw3UaBQ6H",
        "isList": true,
        "isRequired": false,
        "isUnique": null,
        "name": "attachments",
        "relation": null,
      },
      Object {
        "defaultValue": null,
        "displayName": "Field 3",
        "fieldType": "TEXT",
        "fieldTypeAttributes": Object {
          "fieldSize": 256,
          "format": "UNFORMATTED",
        },
        "id": "fldwrjrrd3w4lhDzA",
        "isList": false,
        "isRequired": false,
        "isUnique": null,
        "name": "field_3",
        "relation": null,
      },
    ],
    "id": "tbld5g4XrFhqAKxVT",
    "name": "data",
  },
]
`;
